pkgs/development/python-modules/python-string-utils/default.nix

```diff
@@ -0,0 +1,22 @@
+{
+  lib
+  , buildPythonPackage
+  , fetchPypi
+}:
+
+buildPythonPackage rec {
+  pname = "python-string-utils";
+  version = "1.0.0";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "sha256-3PkGCwPwdkfApgNAjciwP4B/O1SgXG4Z6xRGAlb6wMs=";
+  };
```

Comment:
Missing checkPhase and pythonImportsCheck.

Reply:
tests are nbot bundled via pypi, I added a doCheck = false; with a comment to explain this. I've added the import check